user-interface/src/case-detail/CaseDetailScreen.tsx

```diff
@@ -592,12 +592,13 @@ export default function CaseDetailScreen(props: CaseDetailProps) {
                         id="facet-multi-select"
                         options={getSummaryFacetList(caseDocketSummaryFacets)}
                         onClose={handleSelectedFacet}
-                        onPillSelection={handleSelectedFacet}
                         onUpdateSelection={handleFacetClear}
                         label="Filter by Summary"
                         ariaDescription="Select multiple options. Results will update when the dropdown is closed."
                         aria-live="off"
                         multiSelect={true}
+                        singularLabel="summary"
+                        pluralLabel="summaries"
```

Comment:
Oh, here's a counter example for my earlier claim that we probably wouldn't be encountering edge cases. 😁